plugin/ale/cursor.vim

```diff
@@ -81,7 +95,9 @@ function! ale#cursor#EchoCursorWarning(...)
     let index = s:BinarySearch(loclist, pos[1], pos[2])
 
     if index >= 0
-        call ale#cursor#TruncatedEcho(loclist[index]['text'])
+        let l = loclist[index]
+        let msg = s:GetMessage(l.linter_name, l.type, l.text)
```

Comment:
It is a good idea if we are planning to pass another variables in the future, but I cannot figure out what kind of variables.

Also even if its non noticeable, passing a big dictionnary may take some extra time for processing (a few milliseconds).

So as you wish, should I pass the whole loclist?